docs/can-guides/commitment/recipes/playlist-editor/2-signin.js

```diff
@@ -1,32 +1,41 @@
-const PlaylistVM = can.DefineMap.extend("PlaylistVM", {
-  init: function() {
-    const self = this;
+import { Component } from "//unpkg.com/can@5/core.mjs";
 
-    self.on("googleAuth", function(ev, googleAuth) {
-      self.signedIn = googleAuth.isSignedIn.get();
-      googleAuth.isSignedIn.listen(function(isSignedIn) {
-        self.signedIn = isSignedIn;
-      });
-    });
-  },
-  googleApiLoadedPromise: {
-    default: googleApiLoadedPromise
-  },
-  googleAuth: {
-    get: function(lastSet, resolve) {
-      this.googleApiLoadedPromise.then(function() {
-        resolve(gapi.auth2.getAuthInstance());
+Component.extend({
+  tag: "playlist-editor",
+  view: `
+    {{# if(this.googleApiLoadedPromise.isPending) }}
+      <div>Loading Google API…</div>
+    {{ else }}
+      {{# if(this.signedIn) }}
+      Welcome {{ this.givenName }}! <button on:click="this.googleAuth.signOut()">Sign Out</button>
+      {{ else }}
+      <button on:click="this.googleAuth.signIn()">Sign In</button>
```

Comment:
This line (and 10) should be more indented (here and in the other files).

Reply:
Fixed!